webui/react/src/types.ts

```diff
@@ -199,16 +199,16 @@ export type RecentEvent = {
     name: string;
     date: string;
   };
-}
+};
 
-export type AnyTask = CommandTask | ExperimentTask
+export type AnyTask = CommandTask | ExperimentTask;
 export type RecentTask = AnyTask & RecentEvent;
-export type RecentCommandTask = CommandTask & RecentEvent
-export type RecentExperimentTask = ExperimentTask & RecentEvent
+export type RecentCommandTask = CommandTask & RecentEvent;
```

Comment:
should we make these Error level failures so they break CI test?

Reply:
+ the multiple space rule.

Reply:
yep I like that. I wasn't sure if that was too severe but with auto fix I think that's reasonable.

Reply:
I'll update this PR to make it error level.